resources/assets/coffee/react/_components/icon.coffee

```diff
@@ -17,4 +17,4 @@
 ###
 class @Icon extends React.Component
   render: =>
-    React.DOM.span className: "fa fa-#{@props.name}"
+    React.DOM.span className: "fa fa-#{@props.name} #{@props.className}"
```

Comment:
can't whatever uses this element style it at a parent instead? Most of the attributes in `fa` inherits the parent.